js/core/clusters/clusters.js

```diff
@@ -53,6 +53,11 @@ define([
   return function(config) {
     var lock = false;
 
+    // getter for config.cluster
+    window.cluster = function() {
```

Comment:
`config` is not accessible for security reason, that's why a getter for `config.cluster` makes sense, no technical debt here